base/int.jl

```diff
@@ -482,6 +482,30 @@ julia> trailing_ones(3)
 """
 trailing_ones(x::Integer) = trailing_zeros(~x)
 
+"""
+    ndigits0z2(x::Integer) -> Integer
+
+Efficiently compute `ndigits0z(x, 2)`. This is the number of bits used to represent `x` in
+its binary representation, excluding leading zeros.
+
+`ndigits0z2` is internal and will eventually be replaced by constant propagation.
+
+See also: [`ndigits0z`](@ref), [`ndigits`](@ref).
+
+# Examples
+```jldoctest
+julia> ndigits0z2(4)
+3
+
+julia> ndigits0z2(0)
+0
+
+julia> ndigits0z2(-1)
+64
+"""
+ndigits0z2(x::Integer) = ndigits0z(x, 2)
```

Comment:
Should these methods be moved to `intfuncs.jl` where all other `ndigits*` functions are defined?  This function isn't even used at all in this file, while `intfuncs.jl` is the main consumer (and tests are in `tests/intfuncs.jl`, too)

Reply:
Yep! I placed that method here before I was aware of the `ndigits*` functions.

Reply:
*Actually, this function is used by Core.Compiler and other consumers early in boostrapping so the `::BitInteger` method must stay in int.jl.